EIPS/bls-sig.md

```diff
@@ -0,0 +1,114 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7591
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.